ssl/statem/statem_clnt.c

```diff
@@ -730,32 +730,46 @@ WRITE_TRAN ossl_statem_client_write_transition(SSL_CONNECTION *s)
     }
 }
 
+static int ossl_statem_dtls_client13_use_timer(SSL_CONNECTION *s)
+{
+    OSSL_STATEM *st = &s->statem;
+
+    switch (st->hand_state) {
+    default:
+        break;
+
+    case TLS_ST_CW_ACK:
+        /* Fall through */
+
+    case TLS_ST_OK:
+        return 0;
+    }
+
+    return 1;
+}
+
 int ossl_statem_dtls_client_use_timer(SSL_CONNECTION *s)
 {
     OSSL_STATEM *st = &s->statem;
 
+    if (SSL_CONNECTION_IS_DTLS13(s))
+        return ossl_statem_dtls_client13_use_timer(s);
+
     switch (st->hand_state) {
     default:
         break;
 
-    case TLS_ST_CW_CHANGE:
+    case TLS_ST_CW_KEY_UPDATE:
```

Comment:
This does not make sense to me - we check for DTLS13 above and return. But ST_CW_KEY_UPDATE should not happen in older versions??

Reply:
You are right.. I'll do a fixup.